patches/react-native-webview@13.13.5.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/src/RNCWebViewNativeComponent.ts b/src/RNCWebViewNativeComponent.ts
```

Comment:
Just double checking this is right as I don't do much patching. This looks like a git diff?

Reply:
@edgarkhanzadian looks like this caused an issue though: 
![Uploading Screenshot 2025-05-21 at 12.50.46 PM.png…]()